Gemfile

```diff
@@ -7,115 +7,206 @@ end
 gem 'azure-storage', '~> 0.15.0.preview', require: false
 gem 'azure-storage-blob'
-gem 'bootstrap', '~> 4.6.1'
+# Adds soft delete functionality for models.
+gem 'discard', '~> 1.0'
+# Adds grouping by date/month/etc to queries.
+gem "groupdate", "~> 6.0"
+# Treats attributes like money, which knows about dollars and cents.
+gem "money-rails"
+# Tracks history / audits models.
+gem "paper_trail"
+# Enforces "safe" migrations.
+gem "strong_migrations", "~> 0.7.0"
+
+##### JAVSCRIPT/CSS/ASSETS #######
+
+# Bootstrap is a library for HTML, CSS and JS.
+gem 'bootstrap', '~> 4.6.0'
+# Displays a date range picker, i.e. a way for a user to select a start and end date in
+# a single widget.
 gem 'bootstrap-daterangepicker-rails'
+# Delivers assets for a superpowered dropdown using Bootstrap.
 gem 'bootstrap-select-rails'
-gem "bugsnag"
+# Creates JavaScript graphs.
 gem "chartkick"
+# Installs fonts for Rails frontend.
+gem "font-awesome-rails"
+gem "font-ionicons-rails"
+# A jQuery calendar widget.
+gem "fullcalendar-rails"
+# jQuery framework (DOM methods, Ajax, chaining, etc.)
+gem "jquery-rails"
+gem "jquery-ui-rails"
+# JavaScript date handling functions
+gem "momentjs-rails"
+# SASS CSS framework (nested selectors, variables, etc.)
+gem "sass-rails"
+# Toast (non-blocking notification) library.
+gem "toastr-rails"
+# JavaScript minified, used in asset compilation.
+gem "uglifier", ">= 1.3.0"
+# JavaScript bundler.
+gem 'webpacker', '~> 5.0'
+# Used to verify that the user is a human.
+gem "recaptcha"
+
+##### VIEWS/CONTROLLERS #####
+
+# Adds easy links to add or remove associations in a form (e.g. line items)
 gem "cocoon"
+# Adds filter support to models and views.
+gem "filterrific"
+# Generates JSON structures via a builder interface.
+gem "jbuilder"
+# Pagination of models for use in views.
+gem "kaminari"
+# Reduces boilerplate HTML code when writing forms.
+gem "simple_form"
+# Generate PDFs as views.
+gem "prawn-rails"
+
+##### ADDITIONAL FUNCTIONS #####
+
+# External service that monitors exceptions and reports them to Slack or e-mail.
+gem "bugsnag"
+# Runs jobs asynchronously using the database as a queue.
 gem 'delayed_job_active_record'
+# UI for showing job status.
 gem 'delayed_job_web'
-gem "devise", '>= 4.7.1'
+# Enforces secure passwords for users via policies.
 gem 'devise-secure_password', '~> 2.0'
-gem 'discard', '~> 1.2'
+# Sends invitations via e-mail for new users.
 gem "devise_invitable"
+# Environment variable and configuration management.
 gem "dotenv-rails"
-gem "filterrific"
+# Feature flagging and management.
 gem "flipper"
 gem "flipper-active_record"
 gem "flipper-ui"
-gem "font-awesome-rails"
-gem "font-ionicons-rails"
-gem "fullcalendar-rails"
+# Calculates latitude and longitude from an address.
 gem "geocoder"
-gem "groupdate", "~> 6.0"
-gem "image_processing"
-gem "jbuilder"
-gem "jquery-rails"
-gem "jquery-ui-rails"
+# JSON Web Token encoding / decoding (e.g. for links in e-mails)
 gem "jwt"
```

Comment:
Are we using JWT? This is one I was pretty sure that we'd be able to delete and remove a bunch of code from. I was fairly sure it is a remnant from when we had 2 apps that sent api calls back and forth.

Reply:
We're using it for the account request e-mails.